adafruit_wiznet5k/adafruit_wiznet5k_socketpool.py

```diff
@@ -191,7 +187,7 @@ def getaddrinfo(  # pylint: disable=redefined-builtin,too-many-arguments,unused-
             raise ValueError("Port must be an integer")
         if not self._is_ipv4_string(host):
             host = self.gethostbyname(host)
-        return [(SocketPoolContants.AF_INET, type, proto, "", (host, port))]
+        return [(SocketPool.AF_INET, type, proto, "", (host, port))]
```

Comment:
The name should not be qualified, as it is not in `def socket()` either.

```suggestion
        return [(AF_INET, type, proto, "", (host, port))]
```